packages/types/src/appearance.ts

```diff
@@ -361,6 +361,12 @@ export type Variables = {
    * The background color for the card container.
    */
   colorBackground?: CssColor;
+
+  /*
+   * An array of colors to create the avatar gradient.
+   * This will take place when a user doesn't provide an avatar image.
+   */
+  colorAvatarGradient?: CssColor[];
```

Comment:
```suggestion
  /*
   * If a user hasn't provided an avatar image, a colorful gradient avatar will be shown as fallback.
   * To customise the gradient,pass an array of CSS colors.
   * @default ['#6C47FF', '#5BC5EF', '#FBD486', '#BEF972', '#FF8F8F'];
   */
  colorAvatarGradient?: CssColor[];
```

Also, let's move it right under `colorInputBackground`